tensorrt_llm/usage/usage_lib.py

```
@@ -590,6 +596,17 @@ def _background_reporter(
         arch_class_name = _extract_architecture_class_name(pretrained_config)
         trtllm_config = _extract_trtllm_config(llm_args)
         features_json = _collect_features(llm_args)
+        try:
+            llm_api_config_json, llm_api_config_meta_json = _collect_llm_api_config_payloads(
+                llm_args
+            )
+        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)